backend/src/services/user.service.ts

```diff
@@ -39,19 +38,20 @@ export class UserService {
     login({ email }: IUser) {
         return User.findOne({ where: { email } }).then(u => {
             const { id, email } = u!
-            return { token: jwt.sign({ id, email }, this._jwtSecret) }
+            return { token: jwt.sign({ id, email }, this._jwtSecret), user: u }
```

Comment:
# To Improve
- Update the syntax to an easy readable use Rbnb rules in the linters